app/models/vm_or_template/scanning.rb

```diff
@@ -40,7 +40,7 @@ def scan(userid = "system", options = {})
     begin
       self.last_scan_attempt_on = Time.now.utc
       save
-      job = Job.create_job("VmScan", options)
+      job = ::Job.create_job("VmScan", options)
```

Comment:
@jrafanie it probably does, and I will, but I initially hesitated because of my lack of understanding over what's really going on here. Currently: "This Job is getting preceded by a :ghost: Job in this namespace at some point in the test suite. Sometimes"